canonical-websites/www.ubuntu.com
# Upgrade the run script to the latest version

## Done

Upgraded to the latest ./run

## QA

- Check out this feature branch
- `./run clean`
- Run the site using the command `./run serve`
- Check that there are no errors
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)

## Issue / Card

Fixes #2243
